portal/templates/admin/admin_base.html

```diff
@@ -26,6 +26,9 @@
 	      {% raw %}<div class="error-message">{{errorCollection.orgs}}</div>{% endraw %}
 	    </div>
   	</div>
+  	<div class="orgs-filter-warning" v-show="siteFilterApplied()" data-template='<div class="popover filter-popover alert-icon" role="tooltip" id=“filterPopover” ><div class="arrow"></div><h3 class="popover-title"></h3><div class="popover-content"></div></div>' data-trigger="hover" data-container="body" data-toggle="popover" data-placement="top" data-content="{{_('Site filter applied')}}">
```

Comment:
>id=“filterPopover”

Can you fix the quotes here? I think they're "Microsoft word" quotes (not plaintext)